builtin/logical/nomad/path_roles.go

```diff
@@ -0,0 +1,160 @@
+package nomad
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathListRoles(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "role/?$",
+
+		Callbacks: map[logical.Operation]framework.OperationFunc{
+			logical.ListOperation: b.pathRoleList,
+		},
+	}
+}
+
+func pathRoles(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "role/" + framework.GenericNameRegex("name"),
+		Fields: map[string]*framework.FieldSchema{
+			"name": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Name of the role",
+			},
+
+			"policy": &framework.FieldSchema{
+				Type:        framework.TypeCommaStringSlice,
+				Description: "Policy name as previously created in Nomad. Required",
+			},
+
+			"global": &framework.FieldSchema{
+				Type:        framework.TypeBool,
+				Description: "Policy name as previously created in Nomad. Required",
```

Comment:
Description needs to be updated.

Reply:
Fixed in 3134c7262. Also added a default.